src/event/entity/ItemSpawnEvent.php

```diff
@@ -26,6 +26,14 @@
 use pocketmine\entity\object\ItemEntity;
 
 /**
+ * Called when an item is spawned or loaded.
```

Comment:
Here I have already mentioned that also when it is loaded

Reply:
Yes, but it's just two words, easily overlooked. The entire rest of the doc comment is detailing ways the item might have been created, which creates the misleading impression that that's the only time this event is called.

(Jury is out on whether it should be getting called for items loaded from disk ...)